sim/druid/claw.go

```diff
@@ -0,0 +1,56 @@
+package druid
+
+import (
+	"time"
+
+	"github.com/wowsims/classic/sim/core"
+)
+
+func (druid *Druid) registerClawSpell() {
+	flatDamageBonus := 115.0
+
+	druid.Claw = druid.RegisterSpell(Cat, core.SpellConfig{
+		SpellCode:   SpellCode_DruidClaw,
+		ActionID:    core.ActionID{SpellID: 9850},
+		SpellSchool: core.SpellSchoolPhysical,
+		DefenseType: core.DefenseTypeMelee,
+		ProcMask:    core.ProcMaskMeleeMHSpecial,
+		Flags:       core.SpellFlagMeleeMetrics | core.SpellFlagAPL | SpellFlagOmen | SpellFlagBuilder,
+
+		EnergyCost: core.EnergyCostOptions{
+			Cost:   45 - 1*float64(druid.Talents.Ferocity),
+			Refund: 0.8,
+		},
+		Cast: core.CastConfig{
+			DefaultCast: core.Cast{
+				GCD: time.Second,
+			},
+			IgnoreHaste: true,
+		},
+
+		DamageMultiplierAdditive: 1 + 0.1*float64(druid.Talents.SavageFury),
+		DamageMultiplier:         1,
+		ThreatMultiplier:         1,
+		BonusCoefficient:         1,
+
+		ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+			baseDamage := flatDamageBonus + spell.Unit.MHWeaponDamage(sim, spell.MeleeAttackPower())
+
+			result := spell.CalcAndDealDamage(sim, target, baseDamage, spell.OutcomeMeleeSpecialHitAndCrit)
+
+			if result.Landed() {
+				druid.AddComboPoints(sim, 1, target, spell.ComboPointMetrics())
+			} else {
+				spell.IssueRefund(sim)
+			}
+		},
+	})
+}
+
+func (druid *Druid) CanClaw() bool {
+	return druid.CurrentEnergy() >= druid.CurrentClawCost()
+}
+
+func (druid *Druid) CurrentClawCost() float64 {
+	return druid.Claw.Cost.GetCurrentCost()
+}
```

Comment:
Maybe I'm just missing it but what are these for?

Reply:
These were leftover pasta from Shred but aren't used at all, will remove them. It looks like the Shred ones aren't used anymore either so will remove those too.